x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/Match.java

```diff
@@ -240,19 +245,19 @@ public Match(
                     name = "operator",
                     type = "keyword",
                     valueHint = { "AND", "OR" },
-                    description = "Boolean logic used to interpret text in the query value."
+                    description = "Boolean logic used to interpret text in the query value. Defaults to OR."
                 ),
                 @MapParam.MapParamEntry(
                     name = "prefix_length",
                     type = "integer",
                     valueHint = { "1" },
-                    description = "Number of beginning characters left unchanged for fuzzy matching."
+                    description = "Number of beginning characters left unchanged for fuzzy matching. Defaults to 0."
                 ),
                 @MapParam.MapParamEntry(
                     name = "zero_terms_query",
                     type = "keyword",
                     valueHint = { "none", "all" },
-                    description = "Number of beginning characters left unchanged for fuzzy matching."
+                    description = "Number of beginning characters left unchanged for fuzzy matching. Defaults to none."
```

Comment:
this is not the right doc string for `zero_terms_query` - it seems to be the same from `prefix_length`.

Reply:
Thanks for the catch! Fixed in a6bb85d0589907d081635e7a80be4e552931cff9